Documentation/network/servicemesh/l7-traffic-management.rst

```diff
@@ -16,11 +16,23 @@ and CiliumClusterwideEnvoyConfig).
 Prerequisites
 #############
 
-* Cilium must be configured with ``kubeProxyReplacement`` as partial
-  or strict. Please refer to :ref:`kube-proxy replacement <kubeproxy-free>`
+* Cilium must be configured with ``kubeProxyReplacement`` as ``true``.
+  Please refer to :ref:`kube-proxy replacement <kubeproxy-free>`
   for more details.
 * The minimum supported Kubernetes version for Ingress is 1.19.
 
+Caveats
+#######
+
+* ``CiliumEnvoyConfig`` resources have only minimal validation performed, and
```

Comment:
Could we add some kind of warning that it is also super easy to just pass incorrect config and have it ingested without proper error response to the user?

Reply:
Well, I did my best on this one. 😄